app/controllers/admin/system_commands_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::SystemCommandsController < Admin::BaseController
+  def index
+  end
+
+  def create
+    return redirect_to [:admin, :system_commands], alert: 'Необходимо указать команду' unless params[:name].present?
+    return redirect_to [:admin, :system_commands], notice: 'Недопустимый тип команды' unless allowed_command?
+    SystemCommand.execute(params[:name])
+    redirect_to [:admin, :system_commands], notice: 'Команда отправлена'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.